src/stores/game.js

```diff
@@ -272,6 +272,8 @@ export const useGameStore = defineStore('game', {
     },
     successfullyJoined(player) {
       this.players.push(cloneDeep(player));
+      // sort players so p0 is at index 0 and p1 is at index 1 (bug repair)
```

Comment:
The code is actually very clear and I don’t think we need a comment at all, it’s redundant.